src/app.rs

```diff
@@ -55,6 +55,7 @@ impl Application {
             "off" => "off".to_owned(),
             level => [
                 format!("vector={}", level),
+                format!("remap={}", level),
```

Comment:
I noticed that log messages weren't being output at all. I think possibly this stopped when we factored remap out into it's own crate?

Adding this line fixed it. Is this the right thing to do?

Reply:
:+1: I think this is right.

Reply:
Yes this is right.